docker-compose.yml

```diff
@@ -5,4 +5,4 @@ services:
       context: .
       dockerfile: server.dockerfile
     ports:
-      - "4242:4242"
+      - "8293:8293"
```

Comment:
Any reason why this was changed? doesn't this change the url we post to from the frontend?